### ISSUE TITLE
Add `accessibilityLanguage` attribute to post text

### ISSUE THREAD

**Issue by VyrCossont** [feditext/feditext#141]
**Is your feature request related to a problem? Please describe.**
Posts don't currently have language info attached, so VoiceOver will assume they're written in the system's default language.

**Describe the solution you'd like**
If we have a `language` attribute for a status object, that should be attached to the post's text, CW, poll options, media alt text, and edit history using the [`accessibilityLanguage`](https://developer.apple.com/documentation/objectivec/nsobject/1615192-accessibilitylanguage), so that VoiceOver knows to read a Spanish post in Spanish, etc.

This [doesn't seem to exist under SwiftUI](https://developer.apple.com/documentation/swiftui/accessible-descriptions) so we'd only be able to do it for UIKit components. We might be able to use [`NSAttributedString.Key.accessibilitySpeechLanguage`](https://developer.apple.com/documentation/foundation/nsattributedstring/key/1620188-accessibilityspeechlanguage) as a workaround where text is stored in attributed strings.

**Comment by VyrCossont** [comment]
The SwiftUI equivalent is setting a view's [`.locale()`](https://developer.apple.com/documentation/swiftui/environmentvalues/locale).

**Comment by VyrCossont** [comment]
This should also apply to the post's CW, poll, and media attachment alt text, if present, as well as the post's edit history (which is the SwiftUI part).

**Comment by VyrCossont** [comment]
UIKit part done.

**Comment by VyrCossont** [comment]
SwiftUI part done.

**Comment by VyrCossont** [comment]
The fine-grained way to do it in SwiftUI is to set [`languageIdentifier`](https://developer.apple.com/documentation/foundation/attributescopes/foundationattributes/3802173-languageidentifier) on the relevant parts of a string. This worked fine in `StatusHistoryEditView` so I didn't use `.locale()`.

**Comment by VyrCossont** [comment]
This was done a while ago. Note that labeling parts of a string as different languages can't be tested on VoiceOver in the simulato, where it will read everything as your current language, but it works as expected on a real device: https://princess.industries/@vyr/statuses/01HYEB5B8FYDYG3P7N5XTDM68Q
